examples/text-generation/utils.py

```diff
@@ -636,6 +636,12 @@ def initialize_model(args, logger):
         if not args.parallel_strategy == "tp"
         else setup_distributed_model_tp(args, model_dtype, model_kwargs, logger, cache_dir)
     )
+    if model.config.model_type == "gpt_bigcode" and args.torch_compile and args.use_flash_attention:
```

Comment:
can we add TODO: since we need to remove it later

Reply:
Removed this WA and updated README.